crates/astria-composer/src/config.rs

```diff
@@ -40,6 +40,9 @@ pub struct Config {
 
     /// Address of the RPC server for execution
     pub execution_url: String,
+
+    /// Private key for the sequencer account used for signing transactions
+    pub private_key: String,
```

Comment:
Please use `zeroize` and `ZeroizeOnDrop` derived on `Config` to override this data when config is dropped.

Also, please add a `#[serde(serialize_with = "serialize_scrambled")]` or something like that that hides the value of the string (maybe except for the first few chars).